internal/test/test.go

```diff
@@ -101,8 +101,22 @@ func run(
 		)
 	}
 
+	var filenames []string
+	if len(args) == 0 {
+		filenames, err := filepath.Glob("**/*_test.cdc")
+		if err != nil {
+			return nil, fmt.Errorf("error loading script files: %w", err)
+		}
+
+		if len(filenames) == 0 {
+			return nil, fmt.Errorf("no test files found matching the pattern *_test.cdc")
```

Comment:
Is this an error? Technically not wrong to find no matching patterns right?

Reply:
Yeah, on the fence about this one.

I see value in an error status because it may be informative to the user (i.e. if somebody misconfigures their CI and no test files run, this prevents the situation where they mistakenly assume that their tests are passing).

The flip side is that like you said, it's not necessarly an error state, strictly speaking... an empty result is still a valid result when performing a search.

Personally, I like the error better because it's fairly harmless but exit code 1 ensures that it won't pass CI.  What do you think?

Reply:
Well if no test files were found, would it not say something like "0 tests ran" somewhere else? If not, we should add that in.

Reply:
Yeah, I can just swap it over if you think it's better to exit successfully when no tests are found.  Previously, it was impossible to encounter such a state (command required valid filename arguments).

Reply:
Yeah, it already displays that message.  I've just removed the error.